var/spack/repos/builtin/packages/libepoxy/package.py

```diff
@@ -19,9 +19,22 @@ class Libepoxy(AutotoolsPackage):
     depends_on('pkgconfig', type='build')
     depends_on('meson')
     depends_on('gl')
+    depends_on('libx11', when='+glx')
+
+    variant('glx', default=True, description='enable GLX support')
 
     def configure_args(self):
         # Disable egl, otherwise configure fails with:
         # error: Package requirements (egl) were not met
         # Package 'egl', required by 'virtual:world', not found
-        return ['--enable-egl=no']
+        args = ['--enable-egl=no']
+
+        # --enable-glx defaults to auto and was failing on PPC64LE systems
+        # because libx11 was missing from the dependences. This explicitly
+        # enables/disables glx support.
+        if '+glx' in self.spec:
+            args.append('--enable-glx=no')
```

Comment:
Shouldn't this be `--enable-glx=yes`???

Reply:
@citibeth thanks for catching my copy-and-paste error! Fixed in the latest commit.